Documentation/openshift-issues.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Common Issues on OpenShift
+weight: 10620
+indent: true
+---
+
+# Common Issues on OpenShift
+
+## Monitoring Ceph Cluster on OpenShift
+
+OpenShift Console uses OpenShift Prometheus for monitoring and populating data in Storage Dashboard. Following troubleshooting guide assumes that it is allowed to monitor the Ceph Cluster.
+
+1. Check if Ceph Cluster monitoring is enabled
+
+    `$ kubectl get cephcluster rook-ceph -n rook-ceph -oyaml`
+
+    ```YAML
+    apiVersion: ceph.rook.io/v1
+    kind: CephCluster
+    metadata:
+    name: rook-ceph
+    namespace: rook-ceph
+    [...]
+    spec:
+    [...]
+    monitoring:
+        enabled: true
+        rulesNamespace: "rook-ceph"
+    [...]
+    ```
+
+    > **Note:** Follow [ceph-monitoring/prometheus-alerts](https://github.com/rook/rook/blob/master/Documentation/ceph-monitoring.md#prometheus-alerts) to enable Ceph Cluster monitoring.
+
+2. Check for label on namespace
+
+    `$ kubectl get namespace rook-ceph --show-labels | grep openshift.io/cluster-monitoring=true`
+
+    ```bash
+    rook-ceph   Active   15h   openshift.io/cluster-monitoring=true
+    ```
+
+    If not present, label the namespace with,
+
+    `$ kubectl label namespace rook-ceph "openshift.io/cluster-monitoring=true"`
+
+3. Check if service monitor is present
+
+    `$ kubectl get servicemonitor -n rook-ceph`
+
+    ```bash
+    NAME                          AGE
+    rook-ceph-mgr                 14h
+    ```
+
+4. Check if ceph-mgr pod is Running
+
+    `$ kubectl get pods -l app=rook-ceph-mgr -n rook-ceph`
+
+    ```bash
+    NAME            READY   STATUS    RESTARTS   AGE
+    rook-ceph-mgr   1/1     Running   0          14h
+    ```
```

Comment:
++ check if the monitoring mgr module is enabled?

Reply:
Is there a case when it is disabled? I thought it's always enabled.
How do I check that?

Reply:
You're right, my bad.

Reply:
But I still believe it's good to have it mentioned in a troubleshooting guide.
Log in to the toolbox and run `ceph mgr module ls | grep -A5 enabled_modules`.